kube-config-files/public-concordances-api-service.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name:  public-concordances-api
+  labels:
+    name:  public-concordances-api
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: public-concordances-api
+  template:
+    metadata:
+      labels:
+        app:  public-concordances-api
+        visualize: "true"
+    spec:
+      containers:
+      - name:  public-concordances-api
+        image:  coco/public-concordances-api:v0.1.6
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 256Mi
+        livenessProbe:
```

Comment:
I think that there is a duplicate declaration of liveness probe and readiness probe, see lines 51 and 55